todo_app/templates/index.html

```diff
@@ -4,14 +4,41 @@
 {% block content %}
   <div class="jumbotron">
     <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
+    <p class="lead">My To-Do app.</p>
   </div>
 
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        <!--<table>-->
+          <!--For loop-->
+        {% for item in items %}
+          <!--<tr><td>-->
+            <p> {{item.id}} | {{item.title}} | {{item.status}} </p>
+          <!--</tr></td>-->
+
+          <a href="/">Update</a>
+          <a href="/">Delete</a>
```

Comment:
I assume these were for experimenting, but since they're not working it's quite confusing UX to leave them here! My personal preference would be to delete them (we can always recover them with Git later!) but we could leave them commented if we are certain we'll come back to them later
```suggestion
```